Monika-After-Story/MonikaModDev
# Fixed a crash when generating the shopping list

Currently `mas_generateShoppingList` uses an undefined var in its `for` loop. Good thing we only have 2 consumables officially, but still need to fix it.

EDIT: Actually, found another issue we missed in #5803, but it's minor, so I don't want to spam 10 fixes PRs. The issue is: RenPy marks a button in the extra menu as selected, because it uses the button's `action` property to determinate if it should be selected. Fixed it via explicitly setting the `selected` property to `False`. There's no reason for the button to be selected anyway.

### Testing:
- Verify that `mas_generateShoppingList` is generating the list and doesn't crash anymore.
- Verify that the `Reset` button in the extra menu is not selected.
